augur/utils.py

```diff
@@ -181,7 +182,9 @@ def load_features(reference, feature_names=None):
 
     else:
         from Bio import SeqIO
-        for feat in SeqIO.read(reference, 'genbank').features:
+        reference_record = SeqIO.read(reference, 'genbank')
+        for feat in reference_record.features:
+            feat.seqid = reference_record.name
```

Comment:
I had to check that `.name` was the right attribute here. It seems like it should be equal to `.id` given we don't use a custom `title2ids` parser, but I think it'd still be clearer to use `.id` since we're assigning it to `.seqid`.

For those playing the home game, `Bio.SeqRecord` (what's returned by `Bio.SeqIO.read()`) has `id`, `name`, and `description` attributes, which can all be different. For FASTA, by default, `id` and `name` are the same (the first word of the FASTA defline, sans `>`) and `description` is the whole defline, sans `>`. This can be changed by providing a `title2ids` function to the parser which splits the defline into the three attributes.